lighthouse/blueprints/cherrypicked_plates.py

```diff
@@ -0,0 +1,99 @@
+import logging
+from http import HTTPStatus
+from typing import Any, Dict, Tuple
+
+from flask import Blueprint, request
+from flask_cors import CORS  # type: ignore
+
+from lighthouse.helpers.plates import (
+    add_cog_barcodes,
+    create_cherrypicked_post_body,
+    get_cherrypicked_samples_records,
+    send_to_ss,
+    update_mlwh_with_cog_uk_ids,
+    find_dart_source_samples_rows,
+    find_samples,
+    query_for_cherrypicked_samples,
+    join_rows_with_samples,
+    map_to_ss_columns,
+)
+
+from lighthouse.constants import FIELD_PLATE_BARCODE
+
+logger = logging.getLogger(__name__)
+
+bp = Blueprint("cherrypicked-plates", __name__)
+CORS(bp)
+
+
+@bp.route("/cherrypicked-plates/create", methods=["POST"])
+def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
+    try:
+        barcode = request.get_json()["barcode"]
+        logger.info(f"Attempting to create a plate in SS from barcode: {barcode}")
+    except (KeyError, TypeError) as e:
+        logger.exception(e)
+        return {"errors": ["POST request needs 'barcode' in body"]}, HTTPStatus.BAD_REQUEST
+
+    try:
+        # get_cherrypicked_samples_records(barcode)
+        # get samples from dart for barcode 1234
+        # dart_samples [destination_barcode, destination_well_index, source_barcode, source_well_index, control (String), root_sample_id, rna_id, lab_id]
+        dart_samples = find_dart_source_samples_rows(barcode)
```

Comment:
Should we handle the case where there are no samples in DART for the barcode, and log an error message (similar to below for the Mongo samples)? I guess this **should** never happen though...